ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -140,7 +140,7 @@ const ConfirmPageContainer = (props) => {
       fromAddress,
       global.ethereumProvider,
     );
-    setCollectionBalance(tokenBalance?.balance?.words?.[0] || 0);
+    setCollectionBalance(tokenBalance.toString() || 0);
```

Comment:
I came across this comment
> `decimals` and `balance` are not converted from `BN`s to `string`s automagically in Firefox (as they are in Chrome) so we need to do this for now.
  >>    decimals: details?.decimals?.toString(10),
       balance: details?.balance?.toString(10),

I wonder if we should do the same thing here 🤔  I haven't looked into this far enough yet 
cc: @adonesky1

Reply:
When I tried base 10, some message was thrown in the console, so I switched to the one without the base. But would like to get some insight into this.

Reply:
hmm. I suppose if we write a test for this we can check it will cover Firefox cases too. Is this something we could add?

Reply:
`BigNumber.toString does not accept any parameters; base-10 is assumed` This is the message in the console when I try `setCollectionBalance(tokenBalance.toString(10) || 0);` it is not exactly an error or warning, but it looks like just a log. I cannot test this in Firefox as the app is not working for Firefox in my local.

Reply:
@digiwand I have updated to `tokenBalance.toString(10)`. As I mentioned above, the message we see on the console for Chrome is not an error or warning but just a log. My thoughts are if this change is not having any adverse effect on Chrome, but it is something needed in Firefox, it would be better to have it in rather than leave it out. What do you think?

Reply:
Nice one looking out for the warning message. I think we should follow it and remove the parameter. 

ref: https://github.com/ethers-io/ethers.js/blob/master/packages/bignumber/src.ts/bignumber.ts#L201

---

Now that I've looked at the code some more it looks like the value in [the comment above](https://github.com/MetaMask/metamask-extension/pull/13788/files#r816206665) refers to `BN` object from the 'ethereumjs-util' library. In this PR, this value appears to be a `BigNumber` type from '@ethersproject/bignumber'. 

BN.toString can accept parameters - [ref](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/bn.js/index.d.ts#L75)
BigNumber.toString does not accept any parameters - [ref](https://github.com/ethers-io/ethers.js/blob/master/packages/bignumber/src.ts/bignumber.ts#L201)